external/storm-hive/src/main/java/org/apache/storm/hive/bolt/HiveBolt.java

```diff
@@ -134,22 +131,16 @@ public void execute(Tuple tuple) {
                     collector.ack(t);
                 tupleBatch.clear();
             }
+        } catch(SerializationError se) {
+            LOG.info("Serialization exception occurred, tuple is acknowledged but not written to Hive.", tuple);
+            collector.ack(tuple);
```

Comment:
Why ack this, when before we would fail it?  I realize replaying this is not going to "fix" the problem, but hiding a failure seems worse.